### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by tobias-trabelsi-sonarsource**
Some remarks after the release is published on artifact hub.

### CODE REVIEW

File: charts/sonarqube-dce/Chart.yaml [hunk 1]
```diff
@@ -19,32 +19,9 @@ maintainers:
...
-      links:
-        - name: New location
-          url: TBD
-    - kind: changed
-      description: "changed external database connection logic"
-    - kind: added
-      description: "added option to use external secret as jwtSecret"
-    - kind: fixed
-      description: "removed initSysctl from application pods"
+      description: "Homoginized Description of the Helm Chart"
```
Comment (ganncamp) [inline charts/sonarqube-dce/Chart.yaml:22]: I don't understand. This is a change log section?

File: charts/sonarqube-dce/README.md [hunk 2]
```diff
@@ -35,7 +34,7 @@ The default login is admin/admin.
 
 ## How to use it
 
-Take some time to read the Deploy on [SonarQube on Kubernetes](https://docs.sonarqube.org/latest/setup/sonarqube-on-kubernetes/) page.
+Take some time to read the Deploy on [SonarQube on Kubernetes](https://docs.sonarqube.org/latest/setup/sonarqube-cluster-on-kubernetes/) page.
```
Comment (ganncamp) [inline charts/sonarqube-dce/README.md:37]:
```suggestion
Take some time to read the Deploy [SonarQube on Kubernetes](https://docs.sonarqube.org/latest/setup/sonarqube-cluster-on-kubernetes/) page.
```

**Review by ganncamp — CHANGES_REQUESTED** [review]
One question, one suggestion. Otherwise LGTM

File: charts/sonarqube-dce/Chart.yaml (hunk 1 above)
Comment (tobias-trabelsi-sonarsource) [reply charts/sonarqube-dce/Chart.yaml:22]: Yep this annotation is used on artifacthub to render the changelog: https://artifacthub.io/packages/helm/sonarqube/sonarqube?modal=changelog

**Review by ganncamp — APPROVED** [review]

**Pull request merged**